Makefile

```diff
@@ -273,3 +275,12 @@ pkg/sysfs/sst_types%.go: pkg/sysfs/_sst_types%.go pkg/sysfs/gen_sst_types.sh
 	$(Q)cd $(@D) && \
 	    KERNEL_SRC_DIR=$(KERNEL_SRC_DIR) $(GO_GEN)
 
+report-licenses:
+	$(Q)mkdir -p $(LICENSE_PATH)
+	$(Q)for cmd in $(IMAGE_DIRS); do \
```

Comment:
nitpick: Would be better to blindly follow the pattern of `\t$(Q)$cmd1 && \\\n\t$cmd2...`, so something like
```
report-licenses:
	$(Q)mkdir -p $(LICENSE_PATH) && \
	for cmd in $(IMAGE_DIRS); do \
	...
```

Reply:
Sure, done.